src/htcdaskgateway/cluster.py

```diff
@@ -25,15 +25,27 @@ def __init__(self, **kwargs):
         self.batchWorkerJobs = []
         self.defaultImage = 'coffeateam/coffea-base-almalinux8:0.7.22-py3.10'
         self.cluster_options = kwargs.get('cluster_options')
+        if 'image_registry' in kwargs:
+            self.image_prefix = kwargs.pop('image_registry')
+        else:
+            self.image_prefix = 'registry.hub.docker.com'
```

Comment:
How about
```self.image_prefix = kwargs.get('image_registry', 'registry.hub.docker.com')```

Reply:
Also why not use `self.image_registry` throughout?

Reply:
So the image_registry keyword argument is not a cluster_option, so it has to get popped off the list or it won't instantiate

Reply:
And the self.image_prefix was just how I thought of it, so I didn't change it. I can for consistency, there is no real reason for not using it throughout

Reply:
Oh, good point about the kwargs.

In that case let's be explicit, how about:

```def __init__(self, image_registry="registry.hub.docker.com", **kwargs)```

Reply:
I can do that, I was initially considering this but I figured a kwarg was pretty ok too